www/src/layouts/index.js

```diff
@@ -16,6 +16,8 @@ const styles = css`
   }
 `;
 
+import SEO from '../seo';
```

Comment:
Can we group this with the other imports?

Reply:
Yup, sorry - didn't catch that when resolving the merge conflict, will update!